ansible/publishing.yaml

```diff
@@ -0,0 +1,83 @@
+---
+# Provision an EFS instance for transformers cache
+- name: EFS instance for transformers cache
+  efs:
+    state: present
+    name: "{{ stack_name }}-transformers-cache"
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    region: "{{ region }}"
+    targets:
+      - {subnet_id: "{{ private_subnet_zone_a }}", security_groups: [ "{{ efs_access_sg }}" ]}
+      - {subnet_id: "{{ private_subnet_zone_b }}", security_groups: [ "{{ efs_access_sg }}" ]}
+      - {subnet_id: "{{ private_subnet_zone_c }}", security_groups: [ "{{ efs_access_sg }}" ]}
+    tags:
+      environent: "{{ environment_type }}"
+      teamDL: "universal.publishing.platform@ft.com"
+      systemCode: "{{ platform }}"
+  register: efs_instance_trans_cache
+
+- name: EFS instance details
+  debug:
+    msg: "EFS details: '{{ efs_instance_trans_cache }}'"
+
+# Provision a EFS instance for factset reader
+- name: EFS instance for factset reader
```

Comment:
Seems the factset-reader will be transformed in factset-uploader. Can you please update to this name?